WooCommerce/Classes/POS/Presentation/Reusable Views/POSSendReceiptModalView.swift

```diff
@@ -0,0 +1,84 @@
+import SwiftUI
+
+struct POSSendReceiptModalView: View {
+    @EnvironmentObject private var posModel: PointOfSaleAggregateModel
+    @Environment(\.dismiss) private var dismiss
+
+    @State private var textFieldInput: String = ""
+
+    var body: some View {
+        ZStack(alignment: .topTrailing) {
+            VStack(alignment: .leading, spacing: 20) {
+                Text(Localization.title)
+                    .font(.largeTitle)
+                    .bold()
+
+                Text(Localization.subtitle)
+                    .font(.headline)
+
+                TextField(Localization.textfieldPlaceholder, text: $textFieldInput)
+                    .keyboardType(.emailAddress)
+                    .textInputAutocapitalization(.none)
+                    .autocorrectionDisabled()
+                    .textFieldStyle(RoundedBorderTextFieldStyle(focused: true))
+                    .padding(.horizontal)
+
+                Button(action: {
+                    Task { @MainActor in
+                        await posModel.sendReceipt(to: textFieldInput)
+                    }
+                }, label: {
+                    HStack(spacing: Constants.buttonSpacing) {
+                        Text(Localization.buttonTitle)
+                            .font(Constants.buttonFont)
+                    }
+                    .frame(minWidth: UIScreen.main.bounds.width / 2)
```

Comment:
Perhaps just use `posModalSizing` rather than adding another place where we rely on the screen bounds?

Reply:
Done! https://github.com/woocommerce/woocommerce-ios/pull/14481/commits/9547ed82bc4d8487fdf9819e99ef20afda3cf68e